### ISSUE TITLE
You have OS X 10.11.6. The application requires OS X 10.12 or later

### ISSUE THREAD

**Issue by Rakki1** [sasq64/chipmachine#52]
Latest release doesn't work on OS X 10.11. I have got IMac (Early 2009) so I cannot update to  OS X 10.12.

Error:
"You have OS X 10.11.6. The application requires OS X 10.12 or later"

**Comment by sasq64** [comment]
Yeah, I need to remember to temporarily remove the current SDK while building... OSX makes it a bit tricky to target older versions. Will try to remember it for the next release.

**Comment by sasq64** [comment]
Try the latest beta at http://apone.org/chipmachine/
It should target 10.10, but I have no way of testing.

**Comment by Rakki1** [comment]
Thank you. 

It doesn't start through GUI (right mouse click -> Open application) --> nothing happens. However; Chipmachine starts from command line.

**Comment by sasq64** [comment]
Looks like an illegal opcode. I don't think I set any specific compiler flags...
